homeassistant/components/lightwave.py

```diff
@@ -0,0 +1,49 @@
+"""
+Support for device connected via Lightwave WiFi-link hub.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/lightwave/
+"""
+import voluptuous as vol
```

Comment:
Make sure to sort using `isort`

Reply:
sort what? Are you referring to the import order? Please be explicit.